src/main/java/sawtooth/sdk/messaging/Stream.java

```diff
@@ -18,11 +18,6 @@
 
```

Comment:
This file must be deleted from here? Because I see a new copy with changed generateId() placed under transaction-processor module.